schema/dataKeys.json

```diff
@@ -437,6 +437,55 @@
                   }
                 }
               },
+              "productInfo": {
+                "type": "object",
+                "properties": {
+                  "productName": {
+                    "type": "string",
+                    "description": "productName in content gateway"
+                  },
+                  "productCode": {
+                    "type": "string",
+                    "description": "productCode in content gateway"
+                  },
+                  "productVersionName": {
+                    "type": "string",
+                    "description": "productVersionName in content gateway"
+                  }
+                }
+              },
+              "starmap": {
+                "type": "array",
+                "items": {
+                  "type": "object",
+                  "properties": {
+                    "name": {
+                      "type": "string",
+                      "description": "Artifact name"
+                    },
+                    "workflow": {
+                      "type": "string",
+                      "description": "Push workflow"
+                    },
+                    "cloud": {
+                      "type": "string",
+                      "description": "Cloud provider's name"
+                    },
+                    "mappings": {
+                      "type": "object",
+                      "description": "Mappings for the given artifact"
+                    },
+                    "billing-code-config": {
+                      "type": "object",
+                      "description": "Billing configuration for the community worklow"
+                    }
+                  }
+                }
+              },
+              "cloudMarketplacesSecret": {
+                "type": "string",
+                "description": "Secret for cloud marketplaces"
```

Comment:
You index this as mapping.cloudMarketplacesSecret in the task (not sure why it is in mapping tbh) but here you have it as mapping.components[0].cloudMarketplacesSecret

Reply:
Good catch about it being inside components. Regarding it being in mapping, I asked the same thing here: https://github.com/konflux-ci/release-service-catalog/pull/719#discussion_r1875903632

But maybe it's ok - we do have registrySecret there as well.

Reply:
Oh, let me fix that

Reply:
Fixed, hope it works now